src/components/ProductGrid/ProductGrid.js

```diff
@@ -44,13 +48,26 @@ export default class ProductGrid extends Component {
     );
   }
 
+  renderPageSizeSelector() {
+    const { classes } = this.props;
+
+    return (
+      <Grid container spacing={24} className={classes.pageSizeContainer}>
+        <Grid item>
+          <PageSizeSelector />
+        </Grid>
+      </Grid>
```

Comment:
Why not just a simple flexbox div, full width on mobile and right align when bigger than mobile?

Reply:
The grid container and items render to divs with `display:flexbox` and I that spacing can be set with a prop.